clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/api/KubernetesApiConverter.groovy

```diff
@@ -895,38 +930,74 @@ class KubernetesApiConverter {
       .endMetadata()
       .withNewSpec()
 
-    if (description.restartPolicy) {
-      podTemplateSpecBuilder.withRestartPolicy(description.restartPolicy)
-    }
+    KubernetesPodSpecDescription podSpec = new KubernetesPodSpecDescription()
+    podSpec = description.podSpec
 
-    if (description.terminationGracePeriodSeconds) {
-      podTemplateSpecBuilder.withTerminationGracePeriodSeconds(description.terminationGracePeriodSeconds)
-    }
+    if ( podSpec ) {
```

Comment:
Ah I see - the old path is below - sounds good. 

Given that we're picking a release candidate for 1.0 today/tomorrow I need to hold off on merging this out of fear that we'll destabilize the code right when we need our release pipeline to work.